### PR TITLE
treewide: remove trailing whitespace

### PR DESCRIPTION
**Pull request by siraben**
###### Motivation for this change
We already do this check in ofborg, might as well apply it to all Nix files.

```ShellSession
$  find . -name "*.nix" -type f -print0 | xargs -0 sed -i '' -E "s/[[:space:]]*$//"
```

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: nixos/tests/systemd-networkd-vrf.nix [hunk 1]
```diff
@@ -162,19 +162,19 @@ in {
     # NOTE: please keep in mind that the trailing whitespaces in the following strings
     # are intentional as the output is compared against the raw `iproute2`-output.
     client_ipv4_table = """
-    192.168.1.2 dev vrf1 proto static metric 100
```
Comment (aadibajpai) [inline nixos/tests/systemd-networkd-vrf.nix:165]: these are intentional, see line 162

Comment (siraben) [reply nixos/tests/systemd-networkd-vrf.nix]: Thanks! Resolved.

**Comment by zowoq** [comment]
```
$  find . -name "*.nix" -type f -print0 | xargs -0 sed -i '' -E "s/[[:space:]]*$//"
```
This doesn't consider the exclusions in https://github.com/NixOS/nixpkgs/blob/master/.editorconfig, lots to these files shouldn't be changed.

**Comment by siraben** [comment]
> This doesn't consider the exclusions in https://github.com/NixOS/nixpkgs/blob/master/.editorconfig, lots to these files shouldn't be changed.

Yep, I'm going through that file now and reverting them to`HEAD~2`.

**Comment by zowoq** [comment]
> Yep, I'm going through that file now and reverting them to HEAD~2.

Might be easier to you `editorconfig-checker` to generate the list of files that need changes as it respects the excludes file?

**Comment by siraben** [comment]
Ah, didn't know about that. Thanks.

**Comment by siraben** [comment]
```ShellSession
$ nix-shell -p editorconfig-checker --run 'editorconfig-checker -disable-indent-size -disable-indent-size -disable-insert-final-newline -disable-indentation' | decolorize | grep -v '^[[:blank:]]' | cut -d':' -f1 | xargs sed -i '' -E "s/[[:space:]]*$//"
```

**Review by jonringer — APPROVED** [review]
LGTM

failures are broken on target branch
```
2 packages failed to build:
apitrace minergate-cli

7 packages built:
deadd-notification-center firehol fusee-interfacee-tk keepass ly nanoblogger wavrsocvt
```

**Pull request merged**